### ISSUE TITLE
use * [R] for recurring parent tasks

### ISSUE THREAD

**Issue by linuxcaffe** [tbabej/taskwiki#79]
<pre>
  - [ ] Do this ToDo
</pre>
is valid vimwiki syntax for a todo, but not a taskwiki task. I'm finding it increasingly hard to believe anyone would be using taskwiki and vimwiki-todo in parallel, so I think we should take it over. Using it to indicate that a task is a recurrence-parent would be helpful, and could trigger syntax highliting for tw_color's color.recurring rule.

**Comment by linuxcaffe** [comment]
it should be possible to create a new task with a leading "- [ ] " but if it's not a recurrent-parent, that should be changed to asterisk on saving.

**Comment by tbabej** [comment]
I don't know why a recurring parent task should get such special treatement. I would much rather adopt:

    * [R] This is a recurring parent task

since recurring parent tasks have status:recurring.

**Comment by tbabej** [comment]
Respective syntax highlighting should be added too, with possible sourcing from TW.

**Comment by tbabej** [comment]
Fixed in master.
